src/blocks/form/captcha/attributes.js

```diff
@@ -0,0 +1,7 @@
+const attributes = {
+	id: {
```

Comment:
Do we really need an ID? Can't we just use JS and get the parent's ID instead?

Reply:
We can